packages/maskbook/src/database/Persona/types.ts

```diff
@@ -17,3 +17,15 @@ export interface Persona extends TypedOmit<PersonaRecord, 'localKey' | 'publicKe
 export interface PersonaWithPrivateKey
     extends Omit<PersonaRecord, 'privateKey'>,
         Required<Pick<PersonaRecord, 'privateKey'>> {}
+
+export interface Provider {
+    internalName: string
+    network: string
+    connected: boolean
+    userId: string | undefined
+    identifier: ProfileIdentifier | undefined
+}
+
+export interface PersonaWithProviders extends Persona {
```

Comment:
Please explicitly re-describe the interface with minimal information needed instead of re-use our internal representation. If your public API depended on the internal API, there will be a refactor hazard.

Reply:
get it!